tfhe/src/core_crypto/commons/numeric/unsigned.rs

```diff
@@ -170,7 +183,21 @@ macro_rules! implement {
             fn wrapping_mul(self, other: Self) -> Self {
                 self.wrapping_mul(other)
             }
-            #[must_use]
+            #[inline]
+            fn wrapping_mul_custom_mod(self, other: Self, custom_modulus: Self) -> Self {
+                if Self::BITS < 128 {
```

Comment:
Shouldn't this be `Self::BITS <= 64`, even if we currently don't have types in between?

Reply:
you are right <= 64 is safer though as you note there is no type in between that is not "exotic"

Reply:
done